internal/gengapic/doc_file.go

```diff
@@ -82,10 +82,15 @@ func (g *generator) genDocFile(pkgPath, pkgName string, year int, scopes []strin
 	p("%s%q", "\t", "strings")
 	p("%s%q", "\t", "unicode")
 	p("")
+	p("%s%q", "\t", "google.golang.org/api/option")
 	p("%s%q", "\t", "google.golang.org/grpc/metadata")
 	p(")")
 	p("")
 
+	p("type clientHookParams struct{}")
```

Comment:
link to wikipage somewhere near here?

Reply:
Since we won't have a wiki page until after this is implemented, can I wait to add that once it's published?

Reply:
made a placeholder: https://github.com/googleapis/google-cloud-go/wiki/Customizing-constructors

feel free to rename etc?

Reply:
Oh, yeah, great idea! Added.